magicctapipe/io/io.py

```diff
@@ -66,6 +59,119 @@
 DEAD_TIME_LST = 7.6 * u.us
 DEAD_TIME_MAGIC = 26 * u.us
 
+def check_input_list(config):
```

Comment:
thanks for adding this function - the consistency checks will likely allow to avoid some problems in the future.
can you also add a corresponding unit test, passing a good and bad examples through the function?

Reply:
For sure, we can add it in the test_io

Reply:
Added a test with some test dictionaries; let me know in case more tests are needed

Reply:
thx, looks thorough enough